pkgs/development/web/nodejs/nodejs.nix

```diff
@@ -97,6 +97,12 @@ in
       ${optionalString enableNpm ''
         mkdir -p $out/share/bash-completion/completions/
         $out/bin/npm completion > $out/share/bash-completion/completions/npm
+        for dir in "$out/lib/node_modules/npm/man/"*; do
```

Comment:
You might be able to just do ```mkdir -p $out/share/ && ln -s $out/lib/node_modules/npm/man $out/share/man```

Reply:
unfortunately is not going to work, because node ships `node(1)` manpage
```
$ exa -T result/share/man
result/share/man
└── man1
   └── node.1.gz
```
and with this PR I want to add the npm remaining manpages from npm
```
$ exa -T result/share/man
result/share/man
├── man1
│  ├── node.1.gz
│  ├── npm-access.1 -> ../../../lib/node_modules/npm/man/man1/npm-access.1
|  ⋮
│  ├── npm.1 -> ../../../lib/node_modules/npm/man/man1/npm.1
│  └── npx.1 -> ../../../lib/node_modules/npm/man/man1/npx.1
├── man5
│  ├── npm-folders.5 -> ../../../lib/node_modules/npm/man/man5/npm-folders.5
|  ⋮
│  └── package.json.5 -> ../../../lib/node_modules/npm/man/man5/package.json.5
└── man7
   ├── npm-coding-style.7 -> ../../../lib/node_modules/npm/man/man7/npm-coding-style.7
   ⋮
   ├── removing-npm.7 -> ../../../lib/node_modules/npm/man/man7/removing-npm.7
   └── semver.7 -> ../../../lib/node_modules/npm/man/man7/semver.7

```